packages/teleport-project-generator-angular/src/index.ts

```diff
@@ -0,0 +1,98 @@
+import { createPlugin as createAngularModuleGenerator } from '@teleporthq/teleport-plugin-module-generator-angular'
+import { createAngularComponentGenerator } from '@teleporthq/teleport-component-generator-angular'
+import { createPostProcessor } from '@teleporthq/teleport-postprocessor-prettier-js'
+import { createComponentGenerator } from '@teleporthq/teleport-component-generator'
+import { createProjectGenerator } from '@teleporthq/teleport-project-generator'
+import { FILE_TYPE } from '@teleporthq/teleport-shared/dist/cjs/constants'
+import { Mapping } from '@teleporthq/teleport-types'
+
+import prettierHTML from '@teleporthq/teleport-postprocessor-prettier-html'
+import { createPlugin as createImportPlugin } from '@teleporthq/teleport-plugin-import-statements'
+
+import { CUSTOM_BODY_CONTENT } from './constants'
+import anuglarProjectMapping from './angular-mapping.json'
+
+export const createPreactProjectGenerator = () => {
+  const prettierJS = createPostProcessor({ fileType: FILE_TYPE.TS })
+  const importStatementsPlugin = createImportPlugin({ fileType: FILE_TYPE.TS })
+
+  const rootModuleGeneratorAngular = createAngularModuleGenerator({ moduleType: 'root' })
```

Comment:
maybe you can drop the Angular in the name here, it feels a bit too verbose

Reply:
Yes, Angular in too many places is causing confusion too 😅